templates/stats/context.tpl

```diff
@@ -110,31 +110,26 @@
 							</pkp-button>
 						</template>
 					</pkp-header>
-					<pkp-table
-						labelled-by="contextDetailTableLabel"
-						:class="tableClasses"
-						:columns="tableColumns"
-						:rows="items"
-					>
-						<template #default="{ row, rowIndex }">
-							<table-cell
-								v-for="(column, columnIndex) in tableColumns"
-								:key="column.name"
-								:column="column"
-								:row="row"
-								:tabindex="!rowIndex && !columnIndex ? 0 : -1"
-							>
-								<template #default v-if="column.name === 'title'">
+					<pkp-table aria-label="{translate key="stats.views"}">
+						<table-header>
```

Comment:
So in this case I think we would use the aria-labelled-by..  which again is not in new table yet, but if you can add it.